### DISCUSSION TITLE
Rust Desk Server

### DISCUSSION THREAD

**Discussion by wouldja** [tteck/Proxmox#399] [Ideas]
Title says it all.  It would be nice addition.

**Comment by nacho-domatix** [comment]
This one I guess? https://rustdesk.com/

Seems useful.

**Reply by wouldja** [reply]
That would be it.

**Reply by CaptainNoSleep** [reply]
Came here specifically to see if there was a script for deployment. Upvoted!

**Comment by ScubyG** [comment]
+1 for this

https://rustdesk.com/docs/en/dev/build/docker/

**Reply by nacho-domatix** [reply]
> +1 for this
> 
> https://rustdesk.com/docs/en/dev/build/docker/

Not with docker please. If you want to use docker is easy from CasaOs or DockerLXC, you have scripts for both of them.

**Reply by buczi11** [reply]
+1 ;)

**Comment by TheVons** [comment]
This would be awesome!

**Comment by ianalexis** [comment]
Use this scrip https://github.com/techahold/rustdeskinstall on debian LXC.
Much better than Docker.
